aboutdialog/technicalinfo.cpp

```diff
@@ -163,7 +163,8 @@ TechnicalInfo::TechnicalInfo()
     QString buildType("Release");
 #endif
 
-    table->add("Version",              LXQT_VERSION);
+    table->add("Version",              LXQT_ABOUT_VERSION);
+    table->add("LXQt-Version",         LXQT_VERSION);
```

Comment:
thats easy - my first few cpp lines since 2003 i guess - so it was a first try - and the comments look promising, nobody yells at me, nobody said: Go away, don't touch cpp ... :D